src/pages/Lobby.js

```diff
@@ -78,6 +78,7 @@ class Lobby extends Component{
         this.state = {
             games: []
         }
+        this.subscription = undefined;
```

Comment:
What is this?

Reply:
oops, remove that line, i forgot to remove it. i was trying to fix the "this.subscription is undefined" error but using the new aws-exports file fixed it.